example/app.go

```diff
@@ -53,28 +57,37 @@ func Run() error {
 		panic(err)
 	}
 
-	ethClient, err := evmClient.NewEVMClient(TestEndpoint, false, AliceKp)
+	ethCfg, err := evmClientConfig.GetConfig(".", "config")
+	if err != nil {
+		panic(err)
+	}
+	ethClient := evmClient.NewEVMClient()
+	err = ethClient.InitializeClient(ethCfg, AliceKp)
```

Comment:
I feel like you should pass in `ethCfg` into `NewEVMClient` and then eliminate `InitializeClient` altogether as it is redundant.

`NewEVMClient` should also initialize the client with the passed in config